java/src/jmri/managers/configurexml/DefaultLogixManagerXml.java

```diff
@@ -160,15 +160,14 @@ public void loadLogixs(Element logixs) {
                 List<Element> logixConditionalList = elem.getChildren("logixConditional");  // NOI18N
                 // add conditionals
                 for (Element lxcond : logixConditionalList) {
-                    if (lxcond.getAttribute("systemName") == null) {    // NOI18N
-                        log.warn("unexpected null in systemName {} {}", lxcond, // NOI18N
-                                lxcond.getAttributes());
+                    String cSysName = lxcond.getAttribute("systemName").getValue();  // NOI18N
```

Comment:
Since this will cause an NPE if there is no attribute `systemName`, I suggest something like (pseudo code here):

```
Attribute systemName = lxcond.getAttribute("systemName");
if (systemName == null) {
  log.warn(...);
  break;
}
int cOrder = Integer.parseInt(...)
x.addConditional(systemName.getValue(), cOrder);
```

Reply:
Done